src/unit.test.ts

```diff
@@ -79,147 +142,220 @@ test("kubernetes server operations", async () => {
-    const cleanupResult = await client.request(
-      {
-        method: "tools/call",
-        params: {
-          name: "cleanup",
-          arguments: {}, // Changed from input to arguments
-        },
-      },
-      CleanupResponseSchema
-    );
-    expect(cleanupResult.content[0].type).toBe("text");
-    const cleanupData = JSON.parse(cleanupResult.content[0].text);
-    expect(cleanupData.success).toBe(true);
+    expect(deletePodResult.content[0].type).toBe("text");
+    const deleteResult = JSON.parse(deletePodResult.content[0].text);
+    expect(deleteResult.status).toBe("deleted");
 
-    // Verify cleanup by listing pods again
-    console.log("Verifying cleanup...");
-    const finalPodsResult = await client.request(
-      {
-        method: "tools/call",
-        params: {
-          name: "list_pods",
-          arguments: {
-            // Changed from input to arguments
-            namespace: "default",
-          },
-        },
-      },
-      ListPodsResponseSchema
-    );
-    const finalPods = JSON.parse(finalPodsResult.content[0].text);
-    console.log(finalPods);
-    // expect(finalPods.pods.some((pod: any) => pod.name === "test-pod")).toBe(
-    //   false
-    // );
-  } finally {
-    // await client.disconnect();  // Re-enabled client disconnect
-  }
+    // Try to verify pod termination, but don't fail the test if we can't confirm it
+    try {
+      let podTerminated = false;
+      const terminationStartTime = Date.now();
+
+      while (!podTerminated && Date.now() - terminationStartTime < 10000) {
+        try {
+          const podStatus = await client.request(
+            {
+              method: "tools/call",
+              params: {
+                name: "describe_pod",
+                arguments: {
+                  name: podName,
+                  namespace: "default"
+                }
+              }
+            },
+            ListPodsResponseSchema
+          );
+          
+          // Pod still exists, check if it's in Terminating state
+          const status = JSON.parse(podStatus.content[0].text);
+          if (status.status?.phase === "Terminating") {
+            podTerminated = true;
+            break;
+          }
+          await sleep(500);
+        } catch (error) {
+          // If we get an error (404), the pod is gone which also means it's terminated
+          podTerminated = true;
+          break;
+        }
+      }
+      
+      // Log termination status but don't fail the test
+      if (podTerminated) {
+        console.log(`Pod ${podName} termination confirmed`);
+      } else {
+        console.log(`Pod ${podName} termination could not be confirmed within timeout, but deletion was initiated`);
+      }
+    } catch (error) {
+      // Ignore any errors during termination check
+      console.log(`Error checking pod termination status: ${error}`);
+    }
+  }, { timeout: 90000 }); // Set timeout to 90 seconds to account for all operations
```

Comment:
```suggestion
  });
```

Can remove and keep it using global test timeout